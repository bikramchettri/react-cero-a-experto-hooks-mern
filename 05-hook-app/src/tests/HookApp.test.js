import React from 'react';
import {shallow} from 'enzyme';
import { HooksApp } from '../HooksApp';

describe('Pruebas en HooksApp ', () => {
    test('Debe de mostrarse correctamente', () => {
        const wrapper = shallow(<HooksApp/>) ;
        expect(wrapper).toMatchSnapshot();
    });
});

