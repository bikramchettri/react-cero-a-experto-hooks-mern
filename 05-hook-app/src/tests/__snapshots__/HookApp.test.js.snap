// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HooksApp  Debe de mostrarse correctamente 1`] = `
<div>
  <h1>
    HooksApp
  </h1>
</div>
`;
